src/www/services_rfc2136_edit.php

```diff
@@ -58,6 +58,9 @@
     $pconfig['usetcp'] = isset($a_rfc2136[$id]['usetcp']);
     $pconfig['usepublicip'] = isset($a_rfc2136[$id]['usepublicip']);
 
+    $pconfig['recordtype'] = $a_rfc2136[$id]['recordtype'];
+    if (!$pconfig['recordtype']) $pconfig['recordtype'] = "both";
```

Comment:
Sorry, if I did not get the point but if there is no recordtype set, you are currently updating A and AAAA records which is reflected by "both".